### ISSUE TITLE
Plex error: Error while decoding stream Invalid data found when processing input

### ISSUE THREAD

**Issue by JasonMillward** [JasonMillward/Autorippr#105]
Copied from https://github.com/JasonMillward/Autorippr/issues/102#issuecomment-158855744

Well, as far as I can tell, your script is working, but something changed with how it's doing encodes in this new version and I have no idea why. I manually compiled Handbrake months ago, so it's never updated, and it shouldn't rely on any other libraries.

My encoding settings are exactly the same, yet Plex can no longer read the videos that are output by Handbrake. Did anything change that would change how Handbrake encodes the videos?

In Plex I'm now getting these errors:

```
Nov 22, 2015 22:57:11 [0x7f65123fe700] ERROR - [Transcoder] Error while decoding stream #0:1: Invalid data found when processing input
Nov 22, 2015 22:57:11 [0x7f65103fa700] ERROR - [Transcoder] [mp3 @ 0x20e7d80] Header missing
Nov 22, 2015 22:57:11 [0x7f651c3fd700] ERROR - [Transcoder] Error while decoding stream #0:1: Invalid data found when processing input
Nov 22, 2015 22:57:11 [0x7f651cbfe700] ERROR - [Transcoder] [mp3 @ 0x20e7d80] Header missing
Nov 22, 2015 22:57:11 [0x7f65263fc700] ERROR - [Transcoder] Error while decoding stream #0:1: Invalid data found when processing input
Nov 22, 2015 22:57:11 [0x7f650fbf9700] ERROR - [Transcoder] [mp3 @ 0x20e7d80] Header missing
Nov 22, 2015 22:57:11 [0x7f65253ff700] ERROR - [Transcoder] Error while decoding stream #0:1: Invalid data found when processing input
Nov 22, 2015 22:57:11 [0x7f65123fe700] ERROR - [Transcoder] [mp3 @ 0x20e7d80] Header missing
```

Here are my compress settings, incase you have any idea:

```yaml
compress:
    # Path to compression app (with trailing slash) in case it is unavailable in your $PATH
    compressionPath: ""

    # File format of compressed video (mkv, mp4, avi)
    format:     "mkv"

    # The compression application to use.
    #   handbrake: Compress the video using Handbrake
    #   ffmpeg:    Compress the video using FFmpeg
    type:       handbrake

    # The scheduling priority of the HandBrake program
    #   -20 is the highest (The task gets top priority)
    #    19 is the lowest  (The task get no priority and runs on spare CPU cycles)
    nice:       15

    # The HandBrake command line options and arguments
    # Configure this to change output quality
    # each line should start with -
    com:
        -   --x264-preset="slow"
        -   --two-pass
        -   --turbo
        -   -q 22
        -   --markers
        -   --x264-tune="film"
        -   --encoder="x264"
        -   -x level=4.1

    # Example FFmpeg command line
    #com:
    #    -   -map 0
    #    -   -c copy
    #    -   -c:v libx264
    #    -   -crf 20
    #    -   -preset medium
```

The videos play fine in VLC, and the only difference between the old and new videos I can tell is there's some new metadata information showing in the codec page. The old videos show no metadata information:

![image](https://cloud.githubusercontent.com/assets/507305/11329669/6e6f8282-916d-11e5-912f-a8f1da4c7578.png)

**Comment by ascendedguard** [comment]
Long story short, I believe a Plex update broke MP3 audio in their transcoder. Comparing some old MKVs to newly encoded ones, it looked like different HandBrakeCLI versions may have been used, which I have absolutely no idea how considering the "new" version being shown was the version I compiled back in March.

I just compiled a new version of HandBrake, set the audio to AAC, and everything appears fine now. Go ahead and close this issue.
